opendevin/runtime/docker/ssh_box.py

```diff
@@ -342,6 +346,31 @@ def setup_user(self):
                 raise Exception(
                     f'Failed to chown home directory for opendevin in sandbox: {logs}'
                 )
+            # check the miniforge3 directory exist
+            exit_code, logs = self.container.exec_run(
+                ['/bin/bash', '-c', '[ -d "/opendevin/miniforge3" ] && exit 0 || exit 1'],
+                workdir=self.sandbox_workspace_dir,
+                environment=self._env,
+            )
+            if exit_code != 0:
+                if exit_code == 1:
+                    raise Exception(
+                        f'OPENDEVIN_PYTHON_INTERPRETER is not usable. Please pull the latest Docker image: docker pull ghcr.io/opendevin/sandbox:main'
```

Comment:
Should it be `ghcr.io/opendevin/sandbox:latest`?

Reply:
The `latest` tag was published in April? https://github.com/OpenDevin/OpenDevin/pkgs/container/sandbox/208078240?tag=latest

Reply:
Yeah it has to be `main` now. Once we release a new version (and update the `latest` tag), we could change this exception message.

Reply:
Sounds good!